Multiple configuration files (#511)
MarketSquare/robotframework-tidy: docs/source/configuration/config_file.rst

passed on the command line and going up to parent directories. It stops when it finds the file or it finds root of the
project (determined by existence of ``.git`` directory) or root of the file system.

> Multiple configuration files are allowed. Robotidy will use closest to source configuration file it can find. You can

Reviewer: ```suggestion
Multiple configuration files are allowed. Robotidy will use a configuration file that is closer to the source file. You can
```
